buildpipeline/DotNet-CoreFx-Trusted-Windows-NoTest.json

```diff
@@ -415,7 +415,7 @@
       "allowOverride": true
     },
     "PB_BuildArguments": {
-      "value": "-buildArch=x64 -Release -- /p:SignType=test /p:RuntimeOS=win10",
```

Comment:
I don't think it's a thing anymore, but way back in the day, we would set up the variables so that if someone tried to run one of the builds, it would actually do something.  In this case that would mean adding PB_SignType with a default value into the definition json, then that default value would get overridden by PipeBuild master when it scheduled a build.

Reply:
I knew this and changed it anyways.